.github/workflows/release.yaml

```diff
@@ -0,0 +1,20 @@
+name: release
+on:
+  push:
+    tags:
+    - 'v*'
+jobs:
+  release:
+    name: Release on GitHub
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Create release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          version=$(echo ${{ github.ref }} | sed -e "s#refs/tags/##g")
+          if echo $version | grep -q -e '-'; then prerelease=-p; fi
```

Comment:
Do I understand correctly that if the TAG contains a `-`, it becomes prelease?

Reply:
@zeroalphat 
Yes.
A normal tag should be formed like "x.y.z" and does not contain hyphens.
A use case of a hyphen is `1.xx.0-rc.1` in the following document.
https://neco.cybozu.wiki/procedure_k8s_upgrade/#cke

cf. https://semver.org/#spec-item-9

Reply:
Thank you for your response.
It helped me to understand.